### ISSUE TITLE
Error reported when Turning TAPO L920

### ISSUE THREAD

**Issue by jcbrownratby** [petretiandrea/home-assistant-tapo-p100#736]
## Version of the integration
3.0.0

## Configuration
HASSIO

Tapo L920 Firmware 1.1.3 Build 231229 Rel 164316
```yaml
[139935952979904] Returned error_code: TapoError.ERR_PARAMS: Request params error
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/commands.py", line 239, in handle_call_service
    response = await hass.services.async_call(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2543, in async_call
    response_data = await coro
                    ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/core.py", line 2580, in _execute_service
    return await target(service_call)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 971, in entity_service_call
    single_response = await _handle_entity_call(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/service.py", line 1043, in _handle_entity_call
    result = await task
             ^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/components/light/__init__.py", line 631, in async_handle_light_on_service
    await light.async_turn_on(**filter_turn_on_params(light, params))
  File "/config/custom_components/tapo/light.py", line 143, in async_turn_on
    await self._set_state(
  File "/config/custom_components/tapo/light.py", line 190, in _set_state
    ).get_or_raise()
      ^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/plugp100/common/functional/tri.py", line 159, in get_or_raise
    raise self.value
plugp100.responses.tapo_exception.TapoException: Returned error_code: TapoError.ERR_PARAMS: Request params error```

## Describe the bug

this started with eitehr 2024.4.1 or 2024.4.2. when tuning on the light using a light card and slecting a coulour i get the error. 

## Debug log

<!-- To enable debug logs check this https://www.home-assistant.io/components/logger/ -->

```text

Add your logs here.

```

**Comment by petretiandrea** [comment]
Closed by [3.1.0](https://github.com/petretiandrea/home-assistant-tapo-p100/tree/3.1.0)

**Comment by jcbrownratby** [comment]
on 3.1.1 now, and error persists, so not closed by 3.1.0, sorry

**Comment by jcbrownratby** [comment]
and upgraded to 3.1.2, still the same
